src/core/transport/chttp2/stream_map.c

```diff
@@ -92,6 +96,41 @@ void grpc_chttp2_stream_map_add(grpc_chttp2_stream_map *map, gpr_uint32 key,
   map->count = count + 1;
 }
 
+void grpc_chttp2_stream_map_move_into(grpc_chttp2_stream_map *src,
+                                      grpc_chttp2_stream_map *dst) {
```

Comment:
Similar to what I've commented elsewhere, I think that dst should come before src in general. Not married to the idea, though.